cmd/api/src/bootstrap/server.go

```diff
@@ -67,11 +67,11 @@ func MigrateGraph(ctx context.Context, db graph.Database, schema graph.Schema) e
 
 // MigrateDB runs database migrations on PG
 func MigrateDB(ctx context.Context, cfg config.Configuration, db database.Database) error {
-	if err := db.Migrate(); err != nil {
+	if err := db.Migrate(ctx); err != nil {
```

Comment:
Quick question about the context timeouts. Does it fail the call if the timeout is hit but activity is still happening? I'm just worried a long running migration that touches a lot of data might timeout here when in reality it is doing work.

Reply:
@mistahj67 confirmed that these shouldn't cause timeouts during long running migrations, so I'm good to approve this.